### ISSUE TITLE
Arrow button is not displaying

### ISSUE THREAD

**Issue by GnanaAjana** [brainhubeu/react-carousel#637]
I tried simple example provided in this [docs](https://brainhubeu.github.io/react-carousel/docs/plugins/plugins). arrow icon is not showing. 

So previous and next image i can't able to move

**Comment by andreynering** [comment]
The same here

Arrows are not showing at all, not matter if using the default arrows or custom ones

**Comment by Wekananda** [comment]
Same here, also all plugins can't be loaded

**Comment by thelittlehawk** [comment]
Have you tried following
`
<Carousel
    arrows
/>
`

**Comment by GnanaAjana** [comment]
May i know how to give custom arrows?

**Comment by helloalexdodd** [comment]
Having the same issue. It looks like only the v1 way is working right now.

**Comment by soly2014** [comment]
same here

**Comment by znmto** [comment]
Also having this issue and using `arrows` staggers all the images into a single view:

![image](https://user-images.githubusercontent.com/1846500/91755964-f6b80480-eb99-11ea-8425-1b166838ad3b.png)

```
          <Carousel arrows>
            <img src="https://placekitten.com/500/300" />
            <img src="https://placekitten.com/500/400" />
            <img src="https://placekitten.com/500/200" />
          </Carousel>
```

**Comment by pklepa** [comment]
> Have you tried following
> `<Carousel arrows />`

I was having the same problem and this solved! I also customized the arrows by overriding its class in CSS.

**Comment by Vladis466** [comment]
I belive thats because the package is defaulting to installing version 1.

**Comment by al13bash** [comment]
@Vladis466 I've explicitly installed version 2 and still had similar issues

**Comment by andreecosta** [comment]
> I belive thats because the package is defaulting to installing version 1.

Yep, you are right. I've installed using npm install @brainhubeu/react-carousel@2.0.1 and I can see the arrows now.
Thanks!

**Comment by ChasinHues** [comment]
> I belive thats because the package is defaulting to installing version 1.

This also solved my issue. Using yarn: 

```
yarn add @brainhubeu/react-carousel@^
```

Using `@^` shows a list of available versions for you to choose from. I chose 2.0.1 (latest).

**Comment by jannikkeye** [comment]
> I thought it might have been an issue with the CSS perhaps, but the arrows are not even being rendered to the DOM

Can confirm this with the latest version. No elements are rendered in the dom.

**Comment by withewolf-dev** [comment]
> > I belive thats because the package is defaulting to installing version 1.
> 
> I'm on latest version (2.0.1) using the approach mentioned in the docs:
> 
> ```
> <Carousel
>   plugins={['arrows']}
> >
>   <img src={imageOne} />
>   <img src={imageTwo} />
>   <img src={imageThree} />
> </Carousel>
> ```
> 
> I thought it might have been an issue with the CSS perhaps, but **the arrows are not even being rendered to the DOM**.

How did Got on the latest version ? If you do npm install @brainhubeu/react-carousel I get v1.19.26

**Comment by CitizenBeta** [comment]
@withewolf-dev change your package.json to this:

```
"dependencies": {
    "@brainhubeu/react-carousel": "^2.0.0",
```

**Comment by Amheklerior** [comment]
I've had the same problem, working on a react/typescript project. 

I was using the following Dependencies:
- `react: "17.0.1"`
- `react-dom: "17.0.1"`
- `@brainhubeu/react-carousel: "2.0.2"`

and devDevendencies: 
- `typescript: "4.1.3"`
- `@types/brainhubeu__react-carousel: "2.0.2"`


Downgrading `react` and `react-dom` versions to **16.8.0** (which seem to be peer dependencies of the **2.0.2** version of the library) solved the issue for me.

**Comment by katiawheeler** [comment]
Any update on this? Also experiencing this issue.

**Comment by T0x1c-Devel0per** [comment]
Hello, you must add this line of code.
`<Carousel <br>
            plugins={[
              {
                resolve: arrowsPlugin,
              }
            ]}`
